### ISSUE TITLE
Floating box hides toolbar

### ISSUE THREAD

**Issue by bph** [WordPress/gutenberg#16507]
**Describe the bug**
It's a similar issue as #16055, where the preceding block covers the block toolbar of the succeeding block. 

**To reproduce**
1. Add a cover block
2. Add a gallery block directly underneath
3. try to make the gallery block align wide via the block toolbar

**Expected behavior**
Instead of underneath, i expect the toolbar be on top of the previous block, partly covering the bottom edge of it. 

This was fixed for paragraph block in issue #16055

Here are two examples from today (6.1)  involving the cover block. 
the heading block, only partially hidden. 
![Screen Shot 2019-07-08 at 3 19 20 PM](https://user-images.githubusercontent.com/39980/60836622-19b47780-a194-11e9-8fb4-bbfbdd093ce1.png)
Gallery Block's toolbar is almost entirely hidden. 

![Screen Shot 2019-07-08 at 3 17 54 PM](https://user-images.githubusercontent.com/39980/60836665-2b961a80-a194-11e9-9928-50b61cd84d63.png)

it seems to be connected to the Cover Block as the gallery's toolbar works well with the paragraph block. 
![Screen Shot 2019-07-08 at 3 22 53 PM](https://user-images.githubusercontent.com/39980/60836749-5d0ee600-a194-11e9-8f9a-35e8e5b6b69a.png)

**Comment by jorgefilipecosta** [comment]
Hi @bph, thank you for reporting this issue. In my tests I'm not able to reproduce it, the toolbar appears above the cover, in my tests:
![image](https://user-images.githubusercontent.com/11271197/61209329-58e14c00-a6f1-11e9-800f-a8cbdbd73672.png)

COuld you provide more details namely: theme used, and any plugins installed related to block editor (e.g.: block suits, plugins that provide additional design settings etc...)?
Thank you in advance for the help!

**Comment by bph** [comment]
You are right, @jorgefilipecosta  in its normal width state the cover block behaves. I tried to recreated exactly the situations again. 
Here is a screen recording. 
https://recordit.co/QHJDeQinDH
It shows that the Cover block in its normal width works well with the gallery block, but once I choose the "Wide Width' or the "Full Width", the Block tool bar of the the following paragraph block disappears. I was using Twenty Nineteen. and deactivated all plugins that also have blocks in them. 

**Second Session**

As I wasn't sure of the 20 block plugins didn't leave any residues behind, I started up a new site and recorded a second session. It seems to be connected to the Wide Width and Full Width also with the paragraph block. But then I also added a gallery block and I moved a few blocks around and it suddenly wasn't always consistently covering the block toolbar any more. 
Here is the video

Third Session
I saved the previous test left the block editor and then got back into the edit screen for this post.
https://recordit.co/WnCMqk0F1l
It again started out from the list screen into the edit screen. The cover image is in the Full width and when selecting the paragraph block underneath the cover block, I wasn't able to see the block toolbar. Then I tried a few other combinations and the was a hit and miss. 
It's not entirely consistent but consistent enough to be of concern.
